tst/broken/quick/Sporadic.tst

```diff
@@ -28,6 +28,8 @@ gap> TestSporadic("HN.2"); # FIXME: sometimes gives "B"
 [ "HN.2" ]
 gap> TestSporadic("HS.2"); # FIXME: sometimes "recognized" as McL.2
 [ "HS.2"]
+gap> TestSporadic("ON.2"); # FIXME: sometimes gives "B"
+[ "ON.2" ]
```

Comment:
Actually this doesn't fail on Traivs CI right now?

But anyway, I can reproduce an occasional failure here locally. And I am not surprised by it either. So, sure, let's do that for now.